src/main/java/DeadlineCommand.java

```diff
@@ -0,0 +1,30 @@
+public class DeadlineCommand extends Command {
+    private String command;
+
+    DeadlineCommand(String command) {
+        this.command = command;
+    }
+
+    @Override
+    public String execute(TaskList tasklist, UI ui) throws DukeEmptyDeadlineException, DukeEmptyDeadlineTimeException {
+        String message = "";
+        try {
+            if (command.split(" ").length == 1) {
```

Comment:
May consider adding else block for easier code readability, have noticed this in other classes as well.